### PR TITLE
Fix lint error of status badge

### PR DESCRIPTION
**Pull request by peaceiris**
We need a new line to use the ignore feature.

cf. #340 Log: https://github.com/sdras/awesome-actions/pull/340/checks?check_run_id=478050290#step:4:8

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,11 @@
   <br>
 </p>
 
-# Awesome Actions [![Awesome](https://cdn.rawgit.com/sindresorhus/awesome/d7305f38d29fed78fa85652e3a63e154dd8e8829/media/badge.svg)](https://github.com/sindresorhus/awesome) <!--lint ignore no-dead-urls--> [![GitHub Actions status | sdras/awesome-actions](https://github.com/sdras/awesome-actions/workflows/Lint%20Awesome%20List/badge.svg)](https://github.com/sdras/awesome-actions/actions?workflow=Lint+Awesome+List)
+# Awesome Actions
+
+[![Awesome](https://cdn.rawgit.com/sindresorhus/awesome/d7305f38d29fed78fa85652e3a63e154dd8e8829/media/badge.svg)](https://github.com/sindresorhus/awesome)
+<!--lint ignore no-dead-urls-->
```
Comment (jessicalostinspace) [inline README.md]: This potentially allows an ignore of the https://github.com/sdras/awesome-actions/actions?workflow=Lint+Awesome+List link, which shouldn't be ignored.

Comment (peaceiris) [reply README.md]: See #232 to get the background of why I put the ignore option. We need to login to GitHub for showing the badge image.

Comment (jessicalostinspace) [reply README.md]: The ignore option is definitely the correct option for the link that is having an issue in this case, even though it is masking the underlying issue that's possibly coming from a linter dependency. Please see my fix in https://github.com/sdras/awesome-actions/pull/344/files which only applies the ignore flag to the first markdown node as opposed to both links. This also maintains the layout of the original badges.

Comment (peaceiris) [reply README.md]: I got it.

**Comment by jessicalostinspace** [comment]
This looks good to me 👍 Is it intentional to move the badges below the header here though?
Screen Shot 2020-03-06 at 12 02 00 AM
Screen Shot 2020-03-06 at 12 04 26 AM

**Comment by peaceiris** [comment]
> Is it intentional to move the badges below the header here though?

I just did not consider it. I put badges next to the title.

**Comment by sdras** [comment]
Thanks!

**Pull request merged**